src/vm/eventpipeconfiguration.cpp

```diff
@@ -482,6 +427,8 @@ EventPipeEventInstance *EventPipeConfiguration::BuildEventMetadataEvent(EventPip
     }
     CONTRACTL_END;
 
+    // FIXME: Per-session tracking whether event metadata has been emitted.
```

Comment:
Maybe I misunderstood, I thought based on the meeting today that this was done now?